README.md

```diff
@@ -333,6 +333,47 @@ Component.prototype.isReactComponent = {};
 
 :bulb: To assign the `props` you can simply say: `this.props = props;`
 
-## 11. State
+## 12. State
 
-## 12. Conditional rendering
+As mentioned, the whole point of making this Component class is to be able to create stateful components.
+So finally, let's add some state.
+
+Setting the initial state of your component is really easy. Just assign an object with some properties to the property `state` on your class.
+Just like with props, this is now accessible through `this.state`.
+
+```jsx
+class Greeting extends React.Component {
+    state = { name: "world" };
```

Comment:
Jeg ser at vi har denne i koden vår:

```
    // Do not rerender if setState is called with null or undefined
    if (state == null) {
      return;
    }
```

Burde vi ikke også sjekke på om state er `undefined`. For dette vil vel kræsje om state aldri blir definert i komponenten?

Reply:
@eirikv Her sjekker man for både null og undefined, da man bruker 2 likhetstegn

Reply:
Ah... jaha, overså det... 😅 Er ikke vandt til 2 likhetstegn.

Reply:
Jeg la til en linje om at det bare er å returnere ut av funksjonen med en gang hvis state er null eller undefined.